src/IotHub/IotHub/Az.IotHub.psd1

```diff
@@ -132,7 +132,7 @@ PrivateData = @{
         ReleaseNotes = '* Release for Azure Stack 2019-03-01-hybrid profile'
```

Comment:
we should update teh release notes saying this is for 2020-09-01-hybrid, mentioning the supported api version is a plus